Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -714,19 +721,25 @@ private RepositoryItemBase CopyRIObject(RepositoryItemBase repoItemToCopy)
                     Reporter.ToLog(eLogLevel.ERROR, string.Format("Error occured during object copy of the item: '{0}', type: '{1}', property/field: '{2}'", this.ItemName, this.GetType(), mi.Name), ex);
                 }
             });
+            
+            if(SetNewGUID)
```

Comment:
that is not enough, there is a reason for the GUID replace which been done using ser/des- this is because we need to replace also GUID's placed in different places as String (like in Flow Control- GoToAction)- please undo all changes and keep the GUID replace as before.